propolis/src/hw/bhyve/pmtimer.rs

```diff
@@ -0,0 +1,39 @@
+use std::sync::Arc;
+
+use crate::dispatch::DispCtx;
+use crate::inventory::Entity;
+use crate::migrate::Migrate;
+
+use erased_serde::Serialize;
+
+pub struct BhyvePmTimer {}
```

Comment:
Does this represent the ACPI Power Management Timer?

Reply:
Aye, it does.